test/test_files/act_pitching_wing_2D/sineWave.txt

```diff
@@ -0,0 +1,401 @@
+0.000000 4.000000
```

Comment:
Silly but the style of naming for this file doesn't match the others in this directory. Make it `sine_wave.txt` ?

Reply:
done